### ISSUE TITLE
Code Fixer Support for Packages.

### ISSUE THREAD

**Issue by BrandonSurowiec** [laravel-shift/workbench-issues#36]
**What would you like to do?**
I'd like to run the Laravel code fixer against my package's `src` folder without setting up my own `.php_cs.laravel` file.

Currently the `php/code-format` task fails with this error:

```
In Finder.php line 590:
                                                
  The "/project/app" directory does not exist.
```

**Comment by jasonmccreary** [comment]
Can you tell me a little more about the project. You said package, is it a Composer package? If so, what is under the `autoload` configuration?

**Comment by BrandonSurowiec** [comment]
Sure. It's a private package that has a `composer.json`. The autoload section looks like this:

```
    "autoload": {
        "psr-4": {
            "Slate\\": "src"
        },
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
```

Directory structure is..

- src/
- tests/
- vendor/
composer.json

**Comment by jasonmccreary** [comment]
I see. Shift should have intelligently determined which to scan and not default to `app`. Going to mark this as a bug. We'll get it cleaned up alter this week.

**Comment by jasonmccreary** [comment]
@jessarcher, Shift should get the _source paths_ based on the project. It's possible we have hardcoded `app` somewhere it should not be or are relying on the paths set within our internal CS Fixer ruleset when we may need to be passing the paths when it's not a Laravel app structure.

**Comment by jessarcher** [comment]
Hi @BrandonSurowiec, we've merged a fix for this and will hopefully deploy soon :slightly_smiling_face:

**Comment by BrandonSurowiec** [comment]
Huzzah! Thank you @jessarcher.

**Comment by jasonmccreary** [comment]
This should be patched in the latest release (0.6.1). Let us know if not.

**Comment by BrandonSurowiec** [comment]
@jasonmccreary @jessarcher  Unfortunately it didn't seem to work.

"The "/project/app" directory does not exist."

Is this my error?

![image](https://user-images.githubusercontent.com/5625680/136090595-e59dbb54-cb02-4bea-b8dd-2a5f683bc278.png)

**Comment by jasonmccreary** [comment]
Strange. We definitely fixed this. Did it rebuild the Docker image? Also, how is your `src` folder mapped in the package's composer.json `autoload` configuration?

**Comment by jasonmccreary** [comment]
@BrandonSurowiec, unfortunately although we did make changes to sniff out the source paths for non-Laravel project, the underlying PHP CS Fixer ruleset still looks for these directories. We'll need to find a way around that for non-Laravel projects.

**Comment by jasonmccreary** [comment]
Okay, we just released 0.6.2 which has new assets which should compensate for this PHP CS Fixer restriction. Tested on a few of my packages, but let us know...
